packages/mobile/src/dappkit/DappKitAccountScreen.tsx

```diff
@@ -64,10 +64,6 @@ class DappKitAccountAuthScreen extends React.Component<Props> {
       Logger.error(TAG, 'No account set up for this wallet')
       return
     }
-    if (!phoneNumber) {
-      Logger.error(TAG, 'No phone number set up for this wallet')
-      return
-    }
```

Comment:
What's the impact for Dapps using DAppKit?
Thinking mainly about ImpactMarket and Ubeswap.

Reply:
I tested on Ubeswap and it worked well! 

For ImpactMarket do you mean the web or the app? Not sure how to test with the app but I can give the web a try

Reply:
Tested with ImpactMarket and connected successfully as well :)